src/index.js

```diff
@@ -1,6 +1,7 @@
 
 import getAllFetchedData from './API';
```

Comment:
I'm not sure how this will work. I haven't seen an import that looks like this before.